package-lock.json

```diff
@@ -33,7 +33,7 @@
         "replace-in-file": "6.3.5",
         "rimraf": "3.0.2",
         "standard": "17.0.0",
-        "tap": "^16.3.0",
+        "tap": "16.3.0",
```

Comment:
While it doesn't make sense to @linhkikuchi or I, it appears that this small change caused build failures and that's why we had to revert it earlier: https://github.com/brave/brave-core-crx-packager/pull/444

Reply:
Good to know, I reinstated that - thanks!